autogpts/forge/forge/sdk/errors.py

```diff
@@ -32,10 +32,6 @@ class UnknownCommandError(AgentException):
     hint = "Do not try to use this command again."
 
 
-class DuplicateOperationError(AgentException):
-    """The proposed operation has already been executed"""
-
-
```

Comment:
?

Reply:
Restored, that's related to https://github.com/Significant-Gravitas/AutoGPT/pull/7030